database/models.go

```diff
@@ -4,6 +4,10 @@ type ParentOrg struct {
 	ID   string `json:"id"`
 	Name string `json:"name"`
 	Logo string `json:"logoUrl"`
+	Description string `json:"description"`
+	Skills []string `json:"skills"`
+	Term  []string `json:"terms"`
```

Comment:
Again, the terms need to be linked to a year. How will you store that?
One way might be to store it as using some keyword like sessions, which would contain strings like `2023: Term 2` and so on.

Reply:
This could be done similarly as Latest Year: CurrentTerm